docs/BuildConfiguration.md

```diff
@@ -29,64 +32,128 @@ Sections follow that describe particular settings in more depth.
+| `transitFeeds`                     | Configure access to data transit data sources  (NeTEx, GTFS).                                                                                                                                                                                                                    | array          | null                    |                                                                                           | 
 | `transitServiceStart`              | Limit the import of transit services to the given *start* date. *Inclusive*. Use an absolute date or a period relative to the day the graph is build. To specify a week before the build date use a negative period like `-P1W`.                                                 | date or period | &minus;P1Y              | _2020&#8209;01&#8209;01, &minus;P1M3D, &minus;P3W_                                        |
 | `transitServiceEnd`                | Limit the import of transit services to the given *end* date. *Inclusive*. Use an absolute date or a period relative to the day the graph is build.                                                                                                                              | date or period | P3Y                     | _2022&#8209;12&#8209;31, P1Y6M10D, P12W_                                                  |
 | `writeCachedElevations`            | If true, writes the calculated elevation data.                                                                                                                                                                                                                                   | boolean        | false                   | see [Elevation Data Calculation Optimizations](#elevation-data-calculation-optimizations) |
-
-This list of parameters in defined in
+| `buildReportDir`                   | Path to directory where the build issue report will be written.                                                                                                                                                                                                                  | uri            | null                    |                                                                                           |
+| `localFileNamePatterns`            | Patterns used in determining the type of input files from their names.                                                                                                                                                                                                           | object         | null                    |                                                                                           |
+This list of parameters in defined in 
 the [BuildConfig.java](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/standalone/config/BuildConfig.java)
 .
 
-## Storage
+## Transit Feeds
```

Comment:
Should we explain here if it is possible to define custom configuration for one data set while not defining other data sets in the `transitFeeds` and just trust on the default scanning?